geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffStoreIntegrationTest.java

```diff
@@ -66,6 +66,30 @@ public void sdiffstore_DifferentKeyType_returnsWrongTypeError() {
         .hasMessageContaining(ERROR_WRONG_TYPE);
   }
 
+  @Test
+  public void sdiffstore_withDifferentKeyTypeAndTwoSetKeys_returnsWrongTypeError() {
+    String diffTypeKey = "{tag1}ding";
+    jedis.set(diffTypeKey, "dong");
+
+    String secondSetKey = "{tag1}secondKey";
+    jedis.sadd(setKey, setMembers);
+    jedis.sadd(secondSetKey, setMembers);
+    assertThatThrownBy(() -> jedis.sdiffstore(destinationKey, diffTypeKey, setKey, secondSetKey))
+        .hasMessageContaining(ERROR_WRONG_TYPE);
+  }
+
+  @Test
+  public void sdiffstore_withTwoSetKeysAndDifferentKeyType_returnsWrongTypeError() {
```

Comment:
This test name might be a little clearer as something like "sdiffstore_withNonSetKeyAsThirdSourceKey_returnsWrongTypeError"